include/experimental/__p2630_bits/submdspan_extents.hpp

```diff
@@ -54,37 +107,72 @@ template <class OffsetType, class ExtentType, class StrideType>
 struct is_strided_slice<
     strided_slice<OffsetType, ExtentType, StrideType>> : std::true_type {};
 
-// Helper for identifying valid pair like things
-template <class T, class IndexType> struct index_pair_like : std::false_type {};
+// Traits for pair like things
```

Comment:
[P3663R3](https://isocpp.org/files/papers/P3663R3.html) just passed wording review today and may affect C++26, depending on the results of the C++ Standard Committee plenary meeting next month.  If P3663 is adopted, this part of the implementation would almost certainly require changes.